newdle/models.py

```diff
@@ -6,6 +9,19 @@
 from newdle.core.util import UTCDateTime, format_dt, parse_dt
 
 
+CODE_ALPHABET = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-'
+
+
+def generate_random_code():
+    """Generate a random newdle code, based on a restricted alphabet."""
+    code_length = current_app.config.get('NEWDLE_CODE_LENGTH', 8)
```

Comment:
since we load the example config before the normal config, using `[]` here is safe (and avoids a default value that is never used)